src/test/java/com/softserveinc/ita/vsaroz/PracticeTest.java

```diff
@@ -29,17 +25,17 @@ public void setUp() {
 
     @Test
     public void verifyAddToCartTest () {
-        product
+        productElement
                 .shouldBe(visible)
                 .click();
-        add
+        addToCartButtonElement
                 .shouldBe(visible)
                 .click();
-        closeModal
+        closeModalButtonElement
                 .shouldBe(disappear)
                 .click();
-        viewShoppingCart.click();
-        String quantity = cartQuantity.getText();
-        System.out.println(quantity);
+        viewShoppingCartButtonElement.click();
+        String quantity = itemsInCart.getText();
+        assertThat(quantity).isNotNull();
```

Comment:
add a message for failure

assertThat(quantity)
.as("Incorrect product quantity in the shopping cart")
.isNotNull();  <- compare with expected number, such verification is incorrect and not valid

Reply:
separate assert from code above